app/routes/teams.rb

```diff
@@ -2,6 +2,18 @@ module ExercismWeb
   module Routes
     # rubocop:disable Metrics/ClassLength
     class Teams < Core
+      get '/teams/search' do
```

Comment:
Same here: I think that this should probably be `GET /teams`, the q= param makes it a filter (and without it we would probably technically want to list all of the public teams)